### PR TITLE
Add MeCabTokenizer config

### PR DESCRIPTION
**Pull request by polm-stability**
This adds a config using the MeCabTokenizer for two-pass tokenization.

It also fixes the permissions on an existing config.

### CODE REVIEW

**Comment by polm-stability** [comment]
Thanks for the feedback! I made this by copying another config and just changing two lines related to the tokenization settings, so I wasn't sure what else needed updating. I think this should be OK now?

File: configs/stable-lm-jp/stable-lm-jp-1b-unidiclite_tok-mc4-cc100_wiki.yml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  "distributed-backend": "nccl",
+  "lr-decay-style": "cosine",
+  "warmup": 0.01,
+  "checkpoint-factor": 1000,
+  # 1 more than checkpoint-factor to avoid skipping evals if `evaluate` fails
+  "eval-interval": 1001,
+  "eval-iters": 10,
+  # early logarithmic save iters:
+  # "extra-save-iters": [0,1,2,4,8,16,32,64,128,256,512,1024,2048],
+  "eval_tasks": [
```
[the remark this answers is not in the document]
Comment (leemengtw) [reply configs/stable-lm-jp/stable-lm-jp-1b-unidiclite_tok-mc4-cc100_wiki.yml:94]: Let's add 

```json
"eval_num_fewshot": [
        3,
        3,
        3
    ],
    "eval_limit": [
        0.25,
        0.25,
        0.25
    ],
``` 

under the `eval_tasks` key to make sure we're using the exact setting (#few shots, sample ratio) when doing evaluation.

**Review by leemengtw — CHANGES_REQUESTED** [review]
Almost done! Let's add 2 additional keys for evaluation setting.

**Comment by polm-stability** [comment]
OK, added!

**Review by leemengtw — APPROVED** [review]
LGTM

**Comment by polm-stability** [comment]
OK, I'll merge this, though let's not start training without further discussion.

**Pull request merged**